### ISSUE TITLE
Will this be updated now that the Sdk 20.08 runtime has hit EOL?

### ISSUE THREAD

**Issue by Mitsunee** [flathub/io.atom.Atom#133]
I'm getting warnings about it every time I run `flatpak update`. I'd love to keep using atom for as long as possible, especially in projects that don't use typescript.

```
Looking for updates…
Info: org.freedesktop.Sdk//20.08 is end-of-life, with reason:
   org.freedesktop.Platform 20.08 is no longer receiving fixes and security updates. Please update to a supported runtime version.
Applications using this runtime:
   io.atom.Atom
Info: org.freedesktop.Platform.GL.default//20.08 is end-of-life, with reason:
   org.freedesktop.Platform 20.08 is no longer receiving fixes and security updates. Please update to a supported runtime version.
```

**Comment by cpba** [comment]
I'll try to look into it when I have some time.

I think I tried to fix this last year, but I ran into issues updating the baseapp because this atom build relies on an old zypak-wrapper version to run the launch script. I had to revert my attempt at a fix because it caused issues in old debian releases.

I see there are still two open wip pull requests from when I was trying to find a workaround, but then I stopped using atom and it became much lower priority.
